crates/sui/src/client_commands.rs

```diff
@@ -1101,6 +1169,71 @@ impl SuiClientCommands {
     }
 }
 
+async fn compile_package(
+    client: &SuiClient,
+    build_config: MoveBuildConfig,
+    package_path: PathBuf,
+    with_unpublished_dependencies: bool,
+    skip_dependency_verification: bool,
+) -> Result<
+    (
+        sui_framework_build::compiled_package::PackageDependencies,
```

Comment:
nit: any issue with just importing this?  Seems strange to single it out as fully qualified.